server/migrations/20230731144843-create-channel-banners.js

```diff
@@ -0,0 +1,25 @@
+exports.up = async function (db) {
+  await db.runSql(`
```

Comment:
You're sorting by `uploaded_at` so you should probably have an index to support that

Reply:
You should also consider how you're going to sort these for users (name? I don't know if uploaded_at really makes sense there) and index to support that as well